tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -414,3 +389,58 @@ def __call__(self, event_index):
     param_scheduler.attach(engine, Events.EPOCH_COMPLETED)
 
     engine.run([0] * 8, max_epochs=10)
+
+
+def test_param_scheduler_with_ema_handler_attach_exception():
+    import torch.nn as nn
+
+    from ignite.handlers import EMAHandler
+
+    data = torch.rand(100, 2)
+    model = nn.Linear(2, 1)
+    trainer = Engine(lambda e, b: model(b))
+    param_name = "ema_decay"
+    save_history = True
+    create_new = True
+
+    ema_handler = EMAHandler(model)
+    ema_handler.attach(trainer, name=param_name, event=Events.ITERATION_COMPLETED)
+    ema_decay_scheduler = PiecewiseLinearStateScheduler(
+        param_name=param_name,
+        milestones_values=[(0, 0.0), (10 * len(data), 0.999)],
+        save_history=save_history,
+        create_new=create_new,
+    )
```

Comment:
I think you do not need to add EMAHandler to check `ValueError`.

Reply:
Can we add a working test with EMAHandler and PiecewiseLinearStateScheduler like in the issue ?

Reply:
Yes sure , thanks for your comments @vfdev-5 !